### PR TITLE
Adds America's health ranking from United Healthcare as a source

### PR DESCRIPTION
**Pull request by joshzarrabi**

### CODE REVIEW

File: frontend/src/data/config/FakeDatasetMetadata.ts [hunk 1]
```diff
@@ -107,6 +102,21 @@ const datasetMetadataList: DatasetMetadata[] = [
     name: "COVID-19 deaths, cases, and hospitalizations by gender and state",
     update_time: "March 2021",
   },
+  {
+    id: "uhc_age",
+    name: "COPD and Diabetes prevelence by age and state",
+    update_time: "2020",
```
Comment (vanshkumar) [inline frontend/src/data/config/FakeDatasetMetadata.ts:108]: Just confirming, this update time is correct, right? ie the united health foundation website uses BRFSS data from 2020?

File: frontend/src/data/config/MetadataMap.ts [hunk 2]
```diff
@@ -74,6 +62,19 @@ const dataSourceMetadataList: DataSourceMetadata[] = [
...
+  {
+    id: "uhc",
+    data_source_name: "America's Health Rankings",
+    data_source_link:
+      "https://www.americashealthrankings.org/explore/annual/measure/Overall_a/state/ALL",
+    geographic_level: "State",
+    demographic_granularity: "Race/ethnicity, age, sex",
+    update_frequency: "Annual",
+    description:
+      "The prevalence (percentage) for diseases broken down by self-reported race/ethnicity and sex at the U.S. and state levels.",
```
Comment (vanshkumar) [inline frontend/src/data/config/MetadataMap.ts:74]: Should say "by self-reported race/ethnicity, sex, and age"

**Review by vanshkumar — APPROVED** [review]

File: frontend/src/data/config/FakeDatasetMetadata.ts (hunk 1 above)
Comment (joshzarrabi) [reply frontend/src/data/config/FakeDatasetMetadata.ts:108]: Should be 2019, my b

File: frontend/src/data/config/MetadataMap.ts (hunk 2 above)
Comment (joshzarrabi) [reply frontend/src/data/config/MetadataMap.ts:74]: right, thanks

**Pull request merged**